infrastructure/cdn-in-a-box/traffic_monitor/Dockerfile

```diff
@@ -20,7 +20,7 @@
 ############################################################
 
 ARG RHEL_VERSION=8
-FROM centos:${RHEL_VERSION} as trafficmonitor
+FROM centos:${RHEL_VERSION} as trafficmonitor-dependencies
```

Comment:
I think these types of changes are best left to a separate body of work, since they don't relate to _"Us[ing] `t3c` for cache configuration in CDN-in-a-Box"_

Reply:
While it's true that  it doesn't *directly* relate to using `t3c` for cache configuration CDN in a Box, it is directly related to one of the changes that this PR involves. 942b4396de reorganizes the cache Dockerfile to avoid unnecessarily rebuilding the layer that installs `dlv`, and if that optimization was applied only to the cache Dockerfile, then the debugging stages across our CiaB Dockerfiles would be inconsistent.

So I'd say that including the changes to non-cache Dockerfiles in 942b4396de leads to less entropy over our CiaB Dockerfiles, not less. For the sake of consistency, I'd be tempted to not even apply that optimization to our cache Dockerfile in this PR if not applying it to the other ones in this PR.

Reply:
Yes I think ideally those optimizations would be made at a later date, but they're small enough that we can probably just squeeze them in here. Alternatively, if you have the time and it's not annoying, you could make the optimizations to the other services and submit that as a separate PR as a prerequisite to this one. I can promise to merge that today, if you so choose and once its checks pass.

Reply:
Opened #5866 for this optimization in the other Dockerfiles, removed it from this PR in 062205cfc4.